<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-NM05SVP" name="com.qtpselenium.cucumber.CucumberProjectTestngRunner" tests="4" failures="3" timestamp="18 Aug 2018 04:24:50 GMT" time="1.095" errors="0">
  <testcase name="runScenario" time="0.028" classname="com.qtpselenium.cucumber.CucumberProjectTestngRunner">
    <failure type="org.assertj.core.api.SoftAssertionError" message="
The following 2 assertions failed:
1) expected:&lt;&quot;[B]&quot;&gt; but was:&lt;&quot;[A]&quot;&gt;
at MyHoliday.i_live(MyHoliday.java:35) expected:&lt;&quot;[B]&quot;&gt; but was:&lt;&quot;[A]&quot;&gt;
2) expected:&lt;[tru]e&gt; but was:&lt;[fals]e&gt;
at MyHoliday.we_are(MyHoliday.java:46) expected:&lt;[tru]e&gt; but was:&lt;[fals]e&gt;
">
      <![CDATA[org.assertj.core.api.SoftAssertionError: 
The following 2 assertions failed:
1) expected:<"[B]"> but was:<"[A]">
at MyHoliday.i_live(MyHoliday.java:35) expected:<"[B]"> but was:<"[A]">
2) expected:<[tru]e> but was:<[fals]e>
at MyHoliday.we_are(MyHoliday.java:46) expected:<[tru]e> but was:<[fals]e>

at org.assertj.core.api.SoftAssertions.assertAll(SoftAssertions.java:155)
at com.qtpselenium.cucumber.MyHoliday.afterScenario(MyHoliday.java:29)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at cucumber.runtime.Utils$1.call(Utils.java:26)
at cucumber.runtime.Timeout.timeout(Timeout.java:16)
at cucumber.runtime.Utils.invoke(Utils.java:20)
at cucumber.runtime.java.JavaHookDefinition.execute(JavaHookDefinition.java:60)
at cucumber.runtime.HookDefinitionMatch.runStep(HookDefinitionMatch.java:14)
at cucumber.runner.TestStep.executeStep(TestStep.java:55)
at cucumber.runner.TestStep.run(TestStep.java:42)
at cucumber.runner.TestCase.run(TestCase.java:53)
at cucumber.runner.Runner.runPickle(Runner.java:44)
at cucumber.api.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:56)
at cucumber.api.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:22)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="0.813" classname="com.qtpselenium.cucumber.CucumberProjectTestngRunner">
    <failure type="org.assertj.core.api.SoftAssertionError" message="
The following 2 assertions failed:
1) expected:&lt;&quot;[B]&quot;&gt; but was:&lt;&quot;[A]&quot;&gt;
at MyHoliday.i_live(MyHoliday.java:35) expected:&lt;&quot;[B]&quot;&gt; but was:&lt;&quot;[A]&quot;&gt;
2) expected:&lt;[tru]e&gt; but was:&lt;[fals]e&gt;
at MyHoliday.we_are(MyHoliday.java:46) expected:&lt;[tru]e&gt; but was:&lt;[fals]e&gt;
">
      <![CDATA[org.assertj.core.api.SoftAssertionError: 
The following 2 assertions failed:
1) expected:<"[B]"> but was:<"[A]">
at MyHoliday.i_live(MyHoliday.java:35) expected:<"[B]"> but was:<"[A]">
2) expected:<[tru]e> but was:<[fals]e>
at MyHoliday.we_are(MyHoliday.java:46) expected:<[tru]e> but was:<[fals]e>

at org.assertj.core.api.SoftAssertions.assertAll(SoftAssertions.java:155)
at com.qtpselenium.cucumber.MyHoliday.afterScenario(MyHoliday.java:29)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at cucumber.runtime.Utils$1.call(Utils.java:26)
at cucumber.runtime.Timeout.timeout(Timeout.java:16)
at cucumber.runtime.Utils.invoke(Utils.java:20)
at cucumber.runtime.java.JavaHookDefinition.execute(JavaHookDefinition.java:60)
at cucumber.runtime.HookDefinitionMatch.runStep(HookDefinitionMatch.java:14)
at cucumber.runner.TestStep.executeStep(TestStep.java:55)
at cucumber.runner.TestStep.run(TestStep.java:42)
at cucumber.runner.TestCase.run(TestCase.java:53)
at cucumber.runner.Runner.runPickle(Runner.java:44)
at cucumber.api.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:56)
at cucumber.api.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:22)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="0.208" classname="com.qtpselenium.cucumber.CucumberProjectTestngRunner"/>
  <testcase name="runScenario" time="0.046" classname="com.qtpselenium.cucumber.CucumberProjectTestngRunner">
    <failure type="org.assertj.core.api.SoftAssertionError" message="
The following 2 assertions failed:
1) expected:&lt;&quot;[B]&quot;&gt; but was:&lt;&quot;[A]&quot;&gt;
at MyHoliday.i_live(MyHoliday.java:35) expected:&lt;&quot;[B]&quot;&gt; but was:&lt;&quot;[A]&quot;&gt;
2) expected:&lt;[tru]e&gt; but was:&lt;[fals]e&gt;
at MyHoliday.we_are(MyHoliday.java:46) expected:&lt;[tru]e&gt; but was:&lt;[fals]e&gt;
">
      <![CDATA[org.assertj.core.api.SoftAssertionError: 
The following 2 assertions failed:
1) expected:<"[B]"> but was:<"[A]">
at MyHoliday.i_live(MyHoliday.java:35) expected:<"[B]"> but was:<"[A]">
2) expected:<[tru]e> but was:<[fals]e>
at MyHoliday.we_are(MyHoliday.java:46) expected:<[tru]e> but was:<[fals]e>

at org.assertj.core.api.SoftAssertions.assertAll(SoftAssertions.java:155)
at com.qtpselenium.cucumber.MyHoliday.afterScenario(MyHoliday.java:29)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at cucumber.runtime.Utils$1.call(Utils.java:26)
at cucumber.runtime.Timeout.timeout(Timeout.java:16)
at cucumber.runtime.Utils.invoke(Utils.java:20)
at cucumber.runtime.java.JavaHookDefinition.execute(JavaHookDefinition.java:60)
at cucumber.runtime.HookDefinitionMatch.runStep(HookDefinitionMatch.java:14)
at cucumber.runner.TestStep.executeStep(TestStep.java:55)
at cucumber.runner.TestStep.run(TestStep.java:42)
at cucumber.runner.TestCase.run(TestCase.java:53)
at cucumber.runner.Runner.runPickle(Runner.java:44)
at cucumber.api.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:56)
at cucumber.api.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:22)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- com.qtpselenium.cucumber.CucumberProjectTestngRunner -->
